src/Frame.java

```diff
@@ -19,56 +22,71 @@ public class Frame extends JFrame implements ActionListener{
 
         c1 = new CardLayout();
         cards = new JPanel();
-        first_panel = new Pane();
+        firstPanel = new Pane(fieldSize);
 
         cards.setLayout(c1);
-        cards.add(first_panel, "First panel");
-        new_game();
+        cards.add(firstPanel, "First panel");
+        newGame(fieldSize);
 
-        first_panel.one_player.addActionListener(this);
-        first_panel.two_players.addActionListener(this);
+        firstPanel.onePlayerButton.addActionListener(this);
```

Comment:
Tutaj tak jak rozmawialiśmy, nie wiem za bardzo jak to rozwiązać w tym przykładzie, i myślę, że nie ma co tutaj kminić dalej nad wzorcami, po prostu więcej uwagi przykujemy gdy będziesz działać ze springiem i MVC (bo na tym się znam), to są wzorce architektowniczne.